tests/cypress/e2e/actions_projects_models/markdown_base_pipeline.js

```diff
@@ -166,30 +166,46 @@ context('Basic markdown pipeline', () => {
 
     describe('Staff can see markdown', () => {
         function checkGuideAndAssetAvailableOnAnnotationView() {
+            // when open job for the first time, guide is opened automatically
             cy.visit(`/tasks/${taskID}/jobs/${jobID}`);
             cy.intercept('GET', `/api/assets/${assetID}**`).as('assetGet');
-            cy.get('.cvat-annotation-header-guide-button').should('exist').and('be.visible').click();
+            cy.get('.cvat-annotation-view-markdown-guide-modal button').contains('OK').click();
             cy.wait('@assetGet');
+
+            // when reopen the job, guide is not opened automatically, but can be opened manually
+            cy.reload();
+            cy.get('.cvat-annotation-header-guide-button').should('not.exist');
```

Comment:
Why do we check that `button` doesnt exist and then the same `button` should exist? 
As far as I can understand  its the way we check that the page is loaded, but maybe its more convinient to check that loading spinner doesnt exist and `.cvat-annotation-header` exists

Reply:
Exactly. 

I believe checking spinner is less reliable, because new spinners may appear in the future, with other functionality and it will fail the test.
So, semantically it is preferable to refer to elements that presents feature we are testing.